### ISSUE TITLE
Packaging command for aws-s3 throws TypeError

### ISSUE THREAD

**Issue by cjfinnell** [fab-spec/fab#245]
When attempting to use `fab pacakge <file> --target=aws-s3`, critical failure occurs:
```
$ npx fab package fab.zip -t aws-s3
      [Packager] 💎 fab package 💎

      [Packager] Loading packager code from @fab/deployer-aws-s3
      [Packager] ✔ Done.
    TypeError: packager.createPackage is not a function
```

Version information:
```
$ grep '@fab' package.json
    "@fab/actions": "^1.0.0-rc.6",
    "@fab/cli": "^1.0.0-rc.6",
    "@fab/deployer-aws-s3": "^1.0.0-rc.6",
    "@fab/input-nextjs": "^1.0.0-rc.6",
    "@fab/plugin-render-html": "^1.0.0-rc.6",
    "@fab/plugin-rewire-assets": "^1.0.0-rc.6",
    "@fab/server": "^1.0.0-rc.6.p1",
```

**Comment by evanderkoogh** [comment]
Hey @cjfinnell, I have just been able to reproduce this with a newly created `create-react-app`.

`npx create-react-app project`
`cd project`
`npx fab init`
`yarn build`
`yarn fab:build`
`yarn add @fab/deployer-aws-s3`
`yarn fab package fab.zip -t aws-s3`

And I got the same error. I'll try to have a look at it over the weekend. Thanks for reporting this!

**Comment by evanderkoogh** [comment]
Hey @cjfinnell, I just realised that `aws-s3` not being able to package is as designed. Packing is only used for server-side components because they need to be wrapped inside an adapter that works with the hosting environment.

But S3 is only used for the static assets, so doesn't actually need to be packaged up. I have added a much more helpful message when you try to do this in the future that explains it a bit better.

Thanks for reporting this and helping us improve our docs/error messaging :)
